src/scripts/main.js

```diff
@@ -1 +1,67 @@
 'use strict';
+
+const successEl = (message) =>
+  `<div data-qa="notification" class="success">${message}</div>`;
+const errorEl = (message) =>
+  `<div data-qa="notification" class="error">${message}</div>`;
+
+const displayMessage = (message) => {
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('beforeend', message);
+};
+
+const promise1 = Promise.race([
+  new Promise((resolve) => {
+    const handleClick = () => {
+      resolve(successEl('First promise was resolved'));
+      document.removeEventListener('click', handleClick);
```

Comment:
You are adding and removing event listeners with the same function. This could lead to errors if the function is modified in the future. It's better to define the function separately and then use it as a handler.